hijack/locale/fr/LC_MESSAGES/django.po

```diff
@@ -13,30 +13,33 @@ msgstr ""
 "Plural-Forms: nplurals=2; plural=(n > 1);\n"
 "X-Language: fr\n"
 "X-Source-Language: en\n"
-"X-Generator: Poedit 2.4.2\n"
+"X-Generator: Poedit 3.4.2\n"
 
-#: contrib/admin/admin.py:35
-msgid "hijack user"
-msgstr "détourner un utilisateur"
+#: contrib/admin/admin.py:64
+msgid "impersonate user"
+msgstr "usurper l'identité d'un utilisateur"
```

Comment:
Is it that particular user (in context e.g. button on a profile)? If so, *détourner l'utilisateur*
If indeterminate (any user): *détourner un utilisateur*

Reply:
```suggestion
msgstr "détourner l'utilisateur"
```